### PR TITLE
Tests and tests

### PR DESCRIPTION
**Pull request by profxj**
Added more tests
Figured out how to deal with plots (.travis edit)

### CODE REVIEW

File: linetools/lists/mk_sets.py [hunk 1]
```diff
@@ -100,8 +102,9 @@ def mk_strong(infil=None, outfil=None):
         outfil = infil
     data.write(outfil, format='ascii.fixed_width')
     print('mk_strong: Wrote {:s}'.format(outfil))
+'''
 
-def mk_hi(infil=None, outfil=None):
+def mk_hi(infil=None, outfil=None, no_stop=False):
```
Comment (nhmc) [inline linetools/lists/mk_sets.py]: Can you make this stop=True instead?

File: linetools/lists/mk_sets.py [hunk 2]
```diff
@@ -129,20 +132,23 @@ def mk_hi(infil=None, outfil=None):
...
-    import pdb
-    pdb.set_trace()
+    if not no_stop:
+        import pdb
+        pdb.set_trace()
     data.write(outfil, format='ascii.fixed_width')
 
-def add_galaxy_lines(outfil, infil=None):
+
+def add_galaxy_lines(outfil, infil=None, no_stop=False):
```
Comment (nhmc) [inline linetools/lists/mk_sets.py]: stop=True again

File: linetools/spectra/tests/test_xspectrum_utils.py [hunk 3]
```diff
@@ -149,6 +149,8 @@ def test_copy(spec):
     assert spec.wavelength[0] == spec2.wavelength[0]
     assert spec.flux[-1] == spec2.flux[-1]
 
+def test_plot(spec):
```
Comment (nhmc) [inline linetools/spectra/tests/test_xspectrum_utils.py]: How about we separate all the tests which require xvfb into their own file? Maybe test_plotting.py?

File: .travis.yml [hunk 4]
```diff
@@ -111,6 +111,11 @@ install:
     #- if [[ $SETUP_CMD == 'test --coverage' ]]; then $PIP_INSTALL coverage coveralls; fi
     - if [[ $SETUP_CMD == 'test --coverage' ]]; then $PIP_INSTALL coverage==3.7.1 coveralls; fi
 
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+
```
Comment (nhmc) [inline .travis.yml]: Can you add a comment here saying that this is required for testing plots?

**Comment by nhmc** [comment]
Thanks for working this out! After addressing the inline comments, please merge.

File: linetools/lists/mk_sets.py (hunk 1 above)
Comment (profxj) [inline linetools/lists/mk_sets.py]: yeah, I almost had.

File: linetools/lists/mk_sets.py (hunk 2 above)
Comment (profxj) [inline linetools/lists/mk_sets.py]: Done

File: linetools/spectra/tests/test_xspectrum_utils.py (hunk 3 above)
Comment (profxj) [inline linetools/spectra/tests/test_xspectrum_utils.py]: I've thought about this.  For the code too.
I think it needs a bit more thought first or a 
commitment to a full Refactor.  Punting for now.

File: .travis.yml (hunk 4 above)
Comment (profxj) [inline .travis.yml]: Done.

**Comment by profxj** [comment]
merging.

**Pull request merged**